### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by toor1245**
Any suggestions we may include to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
 cmake_build/
```
Comment (Mizux) [inline .gitignore:1]: `build*/`
`cmake_build*/` ?

Comment (toor1245) [reply .gitignore:1]: what about *build*/?

**Comment by toor1245** [comment]
I would like to add ignore for VS, for example
```gitignore
.vs/*
out/*
```

**Comment by gchatelet** [comment]
Alright, let's do the following:
```
# Build folders
build/
cmake_build/
cmake-build-*/
out/

# IDEs / CI temp files
.idea/
.vagrant/
.vscode/
.vs/
*.swp
```

**Comment by toor1245** [comment]
@gchatelet @Mizux updated

**Review by Mizux — APPROVED** [review]

**Comment by gchatelet** [comment]
Thx!

**Pull request merged**